### DISCUSSION TITLE
AGS install failed with "gobject-introspection-1.0" not found

### DISCUSSION THREAD

**Discussion by Dolbaeb777K** [end-4/dots-hyprland#275] [Q&A] [answered]
```plain
ERROR:  Dependency "gobject-introspection-1.0" not found, tried pkgconfig and cmake
```

When installing this on openSUSE, this error appeared. I do not know what to do. What can I do?

**Comment by clsty** [comment]
I'm not sure how much I can help you since I'm not using openSUSE.

end-4 uses EndeavourOS and NixOS, so maybe no openSUSE either.

But you'd better provide more information, at least what you were doing when encountering this `ERROR`.

**Reply by Dolbaeb777K** [reply]
I installed hyprland using the command in the terminal "sudo zypper in hyprland". Then, I did everything as it is written in the dots-hyprland installation instructions. I wrote it in the terminal: "./update-ags.sh ", there was a "cmake not found" error, but I solved it. But I couldn't get a new one. I'm trying to reinstall the system and go through the installation procedure again without installing hyprland in advance in the GNOME interface

**Comment by Dolbaeb777K** [comment]
[meson-log.txt](https://github.com/end-4/dots-hyprland/files/14358219/meson-log.txt)

Now, I have this error again (the distribution is also openSUSE, I want hyprland on it)

**Comment by clsty** [comment, marked as answer]
Ok, so it's about AGS installation.

Here's the official instruction: <https://aylur.github.io/ags-docs/config/installation/#from-source>

You may use this instruction to install AGS. And if it fails again, please ask for help at [the repo of AGS](https://github.com/Aylur/ags).
